### PR TITLE
Modular Computer Stuff

### PR DESCRIPTION
**Pull request by Geevies**
* Added a default NTNet Relay IRC channel, for everyone to access and congregate at.
* Removed the user has joined and user has left the channel messages, so you can freely move in the IRC without spamming people.
* Added the IRC client as a default program to the Private enrollment preset, so you don't have to download it seperately.
* Renamed the Computer Configuration Tool to Hardware Configuration Tool.
* Added a basic laptop to the loadout.
* Increased the DDOS program's DDOSing power a bit.
* Added Sudoko to the CE's laptop.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_utility.dm [hunk 1]
```diff
@@ -55,9 +55,14 @@
...
 	path = /obj/item/device/paicard
 
+/datum/gear/utility/laptop
+	display_name = "laptop"
+	path = /obj/item/modular_computer/laptop/preset
+	cost = 3
+
 /datum/gear/utility/wallet
 	display_name = "wallet, orange"
-	path = 	/obj/item/storage/wallet
```
Comment (Karolis2011) [inline code/modules/client/preference_setup/loadout/loadout_utility.dm]: Why extra space?

File: html/changelogs/geeves-laptops.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+delete-after: True
+
+changes:
+  - rscadd: "Added a default NTNet Relay IRC channel, for everyone to access and congregate at."
+  - rscdel: "Removed the user has joined and user has left the channel messages, so you can freely move in the IRC without spamming people."
+  - rscadd: "Added the IRC client as a default program to the Private enrollment preset, so you don't have to download it seperately."
+  - tweak: "Renamed the Computer Configuration Tool to Hardware Configuration Tool."
+  - rscadd: "Added a basic laptop to the loadout."
+  - tweak: "Increased the DDOS program's DDOSing power a bit."
+  - rscadd: "Added Sudoko to the CE's laptop."
```
Comment (mikomyazaki) [inline html/changelogs/geeves-laptops.yml]:
```suggestion
  - rscadd: "Added Sudoku to the CE's laptop."
```

**Comment by Geevies** [comment]
old sprites
image

new sprites
image

File: code/modules/client/preference_setup/loadout/loadout_utility.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/client/preference_setup/loadout/loadout_utility.dm]: that is a removed extra space

File: code/modules/modular_computers/NTNet/NTNRC/conversation.dm [hunk 3]
```diff
@@ -38,7 +41,6 @@ var/global/ntnrc_uid = 0
 	if(!istype(C))
 		return
 	clients.Add(C)
-	add_status_message("[C.username] has joined the channel.")
```
[the remark this answers is not in the document]
Comment (Geevies) [reply code/modules/modular_computers/NTNet/NTNRC/conversation.dm]: Can be re-added when we port it to VueUi.

**Pull request merged**